pkg/noderesourcetopology/cache/store.go

```diff
@@ -198,37 +198,48 @@ func (cnt counter) Len() int {
 
 // podFingerprintForNodeTopology extracts without recomputing the pods fingerprint from
 // the provided Node Resource Topology object.
```

Comment:
I would modify comment to explain what is second return value

Reply:
right! fixed